tests/javascript/validate/spec.js

```diff
@@ -199,5 +212,34 @@ define(['jquery', 'testsRoot/validate/spec-setup', 'jasmineJquery'], function ($
 				document.getElementById('button').click();
 			});
 		});
+
+		it('removeMarking method on message validation', function () {
+			beforeAll(function () {
+				document.getElementById('isvalid-numeric-nan').setAttribute('value', 12345);
+			});
+
+			it('should remove class invalid from element #isvalid-numeric-nan after correcting value', function () {
+				expect(document.getElementById('isvalid-numeric-nan')).toHaveClass('valid');
+				document.getElementById('button').click();
```

Comment:
This line doesn't seem to be of any use since its at the end of the test after all assertions